spec/models/restaurant_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe Restaurant do
+  let(:restaurant) { Restaurant.new(name: "test", url: "http://hoge.test/", description: "testtest") }
+
+  describe "name" do
```

Comment:
@yukimurasawa rspec のテストでは、メソッドがクラスメソッドかインスタンスメソッドかわかるように、`.` または `#` をつけるという作法があります！

参考: http://www.betterspecs.org/jp/

ゆうたのPRにしゅんすけもコメントしてますね！
https://github.com/tamago3keran/shuffle_lunch/pull/80#discussion_r369480114

field の場合はインスタンスとして扱うことになるので `#` をつける方が良いかな。
```suggestion
  describe "#name" do
```

@narukami894 spec マスターのしゅんすけにもチェックしてもらいたいです！

Reply:
@katagiri0306 
レビューありがとうございます🙇‍♂️
バリデーションの追加で不具合がないか確認してキャプチャを撮って添付します！
また、rspecの作法についても確認・修正します！💪